pkg/util/exec/exec.go

```diff
@@ -200,7 +200,12 @@ func (*CommandExecutor) ExecuteCommandWithOutputFile(command, outfileArg string,
 	cmd := exec.Command(command, arg...)
 	cmdOut, err := cmd.CombinedOutput()
 	if err != nil {
-		cmdOut = []byte(fmt.Sprintf("%s. %s", string(cmdOut), string(err.(*exec.ExitError).Stderr)))
```

Comment:
There is another instance of this type cast later in the file as well that I assume would need the same fix